monkestation/code/datums/elements/shoesteps.dm

```diff
@@ -26,7 +28,7 @@
 	INVOKE_ASYNC(src, PROC_REF(toggle_sounds), clicker)
 
 /datum/element/shoesteps/proc/toggle_sounds(mob/living/carbon/clicker)
-	if(do_after(clicker, 1.5 SECONDS))
+	if(do_after(clicker, 1.5 SECONDS, interaction_key = DOAFTER_SOURCE_SHOESTEP_TOGGLE))
```

Comment:
oh, uh, should prolly also do something like `if(!DOING_INTERACTION(clicker, DOAFTER_SOURCE_SHOESTEP_TOGGLE)) return` (paraphrased) just to avoid trying a do_after that'll immediately fail if you click again while in the middle of it